# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/eudev/no-sgx.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

see https://gitlab.alpinelinux.org/alpine/aports/-/issues/15043
we don't have sgx support
--
diff --git a/rules/50-udev-default.rules b/rules/50-udev-default.rules
index ff50cb8..2cbd96e 100644
--- a/rules/50-udev-default.rules
+++ b/rules/50-udev-default.rules
@@ -39,9 +39,6 @@ SUBSYSTEM=="cec", GROUP="video"
 SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="video", MODE="0666"
 SUBSYSTEM=="kfd", GROUP="video", MODE="0666"
 
-SUBSYSTEM=="misc", KERNEL=="sgx_enclave", GROUP="sgx", MODE="0660"
-SUBSYSTEM=="misc", KERNEL=="sgx_vepc", GROUP="sgx", MODE="0660"
-
 # When using static_node= with non-default permissions, also update
 # tmpfiles.d/static-nodes-permissions.conf.in to keep permissions synchronized.
 
